Documentation/installation/k8s-install-helm.rst

```diff
@@ -123,7 +123,6 @@ Install Cilium
             --namespace kube-system \\
             --set eni.enabled=true \\
             --set ipam.mode=eni \\
-            --set egressMasqueradeInterfaces=eth+ \\
```

Comment:
Do we need to re-add this for the tunneling mode alternative instructions in the note below? It seems like that note specifically expects Cilium to be performing the masquerade. With tunneling mode since the network is unaware of the Pod IPs, Cilium must be the one responsible for masquerade.

Reply:
this page only covers the ENI mode, and the ENI mode is our recommnded way in EKS environment. Since we dont have any doc talking about the tunneling mode in the AWS/EKS environment. I dont think we should add it here